typescript-local-storage/data.ts

```diff
@@ -0,0 +1,29 @@
+/* exported todos */
+interface Todo {
+  todoId: string;
+  task: string;
+  isCompleted: boolean;
+}
+
+let todos: Todo[] = [];
+
+function writeTodos(): Todo {
+  const todosJSON = JSON.stringify(todos);
+  localStorage.setItem('todos-storage', todosJSON);
+  console.log('the type of', typeof todosJSON);
+  console.log('todosJSON', todosJSON);
+}
+writeTodos();
```

Comment:
Delete this, we should only call this function when we want to update localStorage